Minor Grammar Change

Rebruiza opened the pull request.

babydoll commented on code/game/machinery/vending.dm:
```diff
@@ -479,7 +479,7 @@
 			var/turf/T = get_turf(src)
 			var/mob/living/carbon/alien/diona/S = new(T)
-			src.visible_message("<span class='notice'>\The [src] makes an odd grinding noise before coming to screeching halt as \a [S.name] slurmps out!</span>")
+			src.visible_message("<span class='notice'>\The [src] makes an odd grinding noise before coming to screeching halt as \a [S.name] slumps out!</span>")
```
a screeching halt

comma commented:
I am pretty sure nymphs indeed slurmp instead of slumping, they don't even have posture worth mentioning.

ghost commented:
doona nymp slurmps out of vendo!!!

comma commented:
Exactly, pretty sure it meant to be a silly sound-evoking word, not a typo.

ghost commented:
It is intended to be "slumrp". Just something silly to evoke the sound, that's all.
Source: I coded that.
_I can't believed I missed the "a" there, though. For shame._

PurpleMartinJCK commented:
You need to confirm if you still want to change it now, knowing it's just onomatopoeia.

ghost commented:
I think you meant to add receptacle, not receptable.

ghost commented:
Also, ree, don't remove my made up words.

comma commented:
👎 aside from adding the missing article. Not everything has to be 200% strict dictionary english, soundwords are ok. Please don't dry the text.

ghost commented:
@FTangSteve Yes.

ghost commented:
ugh fine i'll just fix the article and leave the rest

Techhead0 commented on code/game/machinery/vending.dm:
```diff
@@ -479,7 +479,7 @@
 			var/turf/T = get_turf(src)
 			var/mob/living/carbon/alien/diona/S = new(T)
-			src.visible_message("<span class='notice'>\The [src] makes an odd grinding noise before coming to screeching halt as \a [S.name] slurmps out!</span>")
+			src.visible_message("<span class='notice'>\The [src] makes an odd grinding noise before coming to a halt as \a [S.name] is slurmps out from the receptacle.</span>")
```
"... as a diona is slurmps out ..."?

ghost commented:
@Techhead0 Ready for review.

Commit:
Grammar change

wdwadwadwdwaawdwadawad
Yes
Ill squash this later
Why
Update vending.dm
crash bandicoot
Final

Techhead0 reviewed (dismissed):
Terrible commit name, but fine. Approved.

BlueNexus commented:
Seven orphans died when you pushed a commit called "final"

chaoko99 commented:
Doesn't need a squash. It's one commit.

PurpleMartinJCK reviewed (approved):
Looks ok now

Techhead0 commented:
Wait, this is still open?
